controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -351,9 +352,18 @@ private ReaderGroupConfiguration getRGConfigurationFromRecord(final String scope
                          return eventHelper.addIndexAndSubmitTask(event,
```

Comment:
the event being passed here needs to be updated with correct rg id too.

Reply:
As discussed, the event should have rgId from the config passed by user to avoid overwriting the existing config with a new one passed in createRG API.